File: C#/URI/3224.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        string jon = Console.ReadLine();
        String medico = Console.ReadLine();

        int a = jon.Length;
        int b = medico.Length;

        if (a >= b)
            Console.WriteLine("go");
        else
             Console.WriteLine("no");

    }
}

File: C/URI/3224.c
#include<stdio.h>
int main(){

    char jon[1000], medico[1000];
    

    scanf("%s", jon);
    scanf("%s", medico);

    int a = strlen(jon);
    int b = strlen(medico);

    if(a >= b)
        printf("go\n");
    else
        printf("no\n");


    return 0;
}